static/main.css

```diff
@@ -0,0 +1,3 @@
+html {
+  overflow: hidden;
+}
```

Comment:
Contact and review with a person who makes a base frontend, to not create excessive files.

Reply:
you can't init an empty directory in github, this is a placeholder.